scripts/devenv-builder/manage-vm.sh

```diff
@@ -193,13 +210,15 @@ function action_help {
 function usage {
     local -r script_name=$(basename "$0")
     cat - <<EOF
-${script_name} (create|ip|delete|rm|help) [options]
+${script_name} (create|ip|ssh|console|delete|rm|help) [options]
```

Comment:
Looks like `config` is missing here - can we fix it in this PR?

Reply:
Done